about.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+        integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/aboutpage.css">
+    <link rel="stylesheet" href="/css/aboutheader.css">
+    <link rel="stylesheet" href="/css/footer.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+</head>
+
+<body>
+    <header>
+        <div class="topnav">
+            <nav>
+                <li><i class="fa-brands fa-instagram-square"></i></li>
+                <li> <i class="fa-brands fa-twitter"></i></li>
+                <li><i class="fa-brands fa-facebook"></i></li>
+            </nav>
+        </div>
+        <div class="mainnav">
+            <h1 class="main-h3"> The chance</h1>
+            <nav>
+                <li><a href="/about.html">About</a></li>
+                <li> Contact</li>
+                <li> Sponsor</li>
+                <li>News</li>
+                <button class="main-button"> Buy Ticket now</button>
+            </nav>
+        </div>
+
+
+        <button type="button" class="ham">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+        </button>
+
+        </div>
+
+        <div class="overlayy"></div>
+        <nav class="sideNav">
+            <button type="button" class="menuX">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+                </svg>
+            </button>
+            <ul>
+                <li><a class="menuLink" href="/index.html">Home</a></li>
+                <li><a class="menuLink" href="#projects-container">Contact</a></li>
+                <li><a class="menuLink" href="#about-container">Sponsort</a></li>
+                <li><a class="menuLink main-button" href="#footer">Buy Tickets</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero-section background-imagee ">
```

Comment:
- [ ] Kindly be noticed that the background is transparent or has the same color/pattern as the top section background.